parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -94,6 +94,9 @@ impl_opaque_keys! {
 #[cfg(feature = "state-trie-version-1")]
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
+	// Note: "statemine" is the legacy name for this chain. It has been renamed to
```

Comment:
I wish there was a way to have a more concrete and delicate plan here, similar to https://github.com/paritytech/substrate/issues/13976#issuecomment-1568538473

Reply:
Yeah, I was thinking the same. There's not really a good way to change it except to leave it, hope enough tools update to support the change, and then break it.